examples/policy/cilium/README.md

```
@@ -0,0 +1,624 @@
+    --from-file=ca.crt=/tmp/gateway-ca.crt \
+    --dry-run=client -o yaml | kubectl apply -f -
+
+  shred -u /tmp/gateway-ca.key /tmp/gateway-ca.crt
+  ```
+
+  See §7.5 for the rationale (why this is admin-created out-of-band, security considerations, rotation).
+
+## 3. CUJ coverage
+
+The CUJs that motivate this example live in 
+| CUJ | Primitive | Manifest | Coverage |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)